hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/recovery/ReconOMMetadataManager.java

```diff
@@ -47,4 +50,59 @@ public interface ReconOMMetadataManager extends OMMetadataManager {
    * @return true if OM Tables are initialized, otherwise false.
    */
   boolean isOmTablesInitialized();
+
+  /**
+   * Return a list of volumes owned by a given user; if user is null, returns
+   * all volumes.
+   *
+   * This method can be optimized by using username as a filter.
+   *
+   * @param startKey the start volume name determines where to start listing
+   * from, this key is excluded from the result. Returns all the volumes if
+   * it's null.
+   * @param maxKeys the maximum number of volumes to return.
+   * @return volumes with starting from <code>startKey</code> limited by
+   *         <code>maxKeys</code>
+   */
+  List<OmVolumeArgs> listVolumes(String startKey,
+                                 int maxKeys) throws IOException;
+
+  /**
+   * Return all volumes in the file system.
+   * @return all the volumes from the OM DB.
```

Comment:
Pls update the comment if this API is bounded by limit.

Reply:
This is referring to `ReconOMMetadataManager#listVolumes()` which is only used in `RootEntityHandler`. The UI will use `ReconOMMetadataManager#listVolumes(String startKey, int maxKeys)`. The reason is to prevent behaviour change in `RootEntityHandler`. So this method will still return all the volumes in the file system.